src/stream/src/task/stream_manager.rs

```diff
@@ -673,46 +664,28 @@ impl LocalStreamManagerCore {
                 .map(|id| (actor.actor_id, *id))
                 .collect_vec();
             update_upstreams(&self.context, &down_id);
-
-            // Add remote input channels.
-            let mut up_id = vec![];
-            for upstream_id in actor.get_upstream_actor_id() {
-                if !local_actor_ids.contains(upstream_id) {
-                    up_id.push((*upstream_id, actor.actor_id));
-                }
-            }
-            update_upstreams(&self.context, &up_id);
```

Comment:
No channel is required on the downstream node for remote exchange so that we can remove this logic.